sample-apps/react-native/dogfood/src/components/NavigationHeader.tsx

```diff
@@ -19,12 +19,13 @@ import { Button } from './Button';
 export const NavigationHeader = ({ route }: NativeStackHeaderProps) => {
   const videoClient = useStreamVideoClient();
   const { t } = useI18n();
-  const username = useAppGlobalStoreValue((store) => store.username);
+  const userName = useAppGlobalStoreValue((store) => store.userName);
+  const userId = useAppGlobalStoreValue((store) => store.userId);
   const appStoreSetState = useAppGlobalStoreSetState();
 
   const logoutHandler = () => {
     Alert.alert(
-      `Sign out as ${username}`,
+      `Sign out as ${userName || userId}`,
```

Comment:
lets only use username here

Reply:
If the name isn't provided for custom login user, the name will be empty.

Reply:
Done